### ISSUE TITLE
Inconsistent setting names

### ISSUE THREAD

**Issue by Kriechi** [python-hyper/hyperframe#29]
Hi,

I noticed a slight inconsistency with the setting variable names:
https://github.com/python-hyper/hyperframe/blob/4fe1a74d026f5c75c569979eadb663dcac2546a0/hyperframe/frame.py#L353-L356

`SETTINGS_MAX_FRAME_SIZE` and `SETTINGS_MAX_HEADER_LIST_SIZE` are the only ones prefixed with `SETTINGS_*`. Is there any reason why, or can we unify all the names here?

**Comment by Lukasa** [comment]
Yeah, this is a problem I've been aware of for a little while: presumably I suffered some kind of temporary lapse in judgement when I added those later variable names. You'll notice that [hyper-h2's version of those variable names does not have this problem](http://python-hyper.org/h2/en/stable/api.html#known-settings).

We absolutely can unify the names, but I'm not really in any rush to do it because it's either a semver-major change or it introduces duplication/aliasing into the class. For people using hyper-h2 they shouldn't have this problem, because they can use the aliases hyper-h2 provides. For people using hyperframe we should fix it, but I'm inclined to let it sit until we have some other semver-major changes we need to land.

Does that suit you @Kriechi?
